File: library/src/main/java/sk/vander/library/BaseGraph.java
package sk.vander.library;


import sk.vander.library.navigation.activity.ActivityScreenSwitcher;

/**
 * Created by arashid on 24/06/16.
 */
public interface BaseGraph {
  AppContainer appContainer();
  ActivityHierarchyServer activityHierarchyServer();
  ActivityScreenSwitcher activityScreenSwitcher();
}

File: library/src/main/kotlin/sk/vander/library/BaseGraph.kt
package sk.vander.library


import sk.vander.library.navigation.activity.ActivityScreenSwitcher

/**
 * Created by arashid on 24/06/16.
 */
interface BaseGraph {
  fun appContainer(): AppContainer
  fun activityHierarchyServer(): ActivityHierarchyServer
  fun activityScreenSwitcher(): ActivityScreenSwitcher
}
